### PR TITLE
create database

### PR DESCRIPTION
**Pull request by matthewdking**
relates #3 #4

### CODE REVIEW

File: src/database/db_build.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+CREATE TABLE questions(
+  id          SERIAL    PRIMARY KEY,
+  category    TEXT      NOT NULL,
+  question    TEXT      NOT NULL
+);
+
+CREATE TABLE singing(
+  id        SERIAL        PRIMARY KEY,
+  name      VARCHAR(20)   NOT NULL,
```
Comment (sajedazoabi) [inline src/database/db_build.sql]: I think we should join the singing table to the mentors' table using the id column, so there is no need for the name column.

Comment (matthewdking) [reply src/database/db_build.sql]: :+1: good spot I think we could call the column mentor_id and have it equal the id in the mentors table. Thoughts?

**Review by Wyrd00 — CHANGES_REQUESTED** [review]
- can ya change the image folder --> assets folder?

**Review by Wyrd00 — APPROVED** [review]
👍

**Review by sajedazoabi — APPROVED** [review]

**Review by Karyum — APPROVED** [review]
:+1:

**Pull request merged**